megalinter/MegaLinter.py

```diff
@@ -501,11 +506,14 @@ def list_files_all(self):
         return all_files
 
     def list_git_ignored_files(self):
-        repo = git.Repo(os.path.realpath(self.github_workspace))
-        ignored_files = repo.git.execute("git status --ignored")
-        ignored_files = list(
-            map(lambda x: x + "**" if x.endswith("/") else x, ignored_files)
-        )
+        dirpath = os.path.realpath(self.github_workspace)
+        repo = git.Repo(dirpath)
+        # ignored_files = repo.git.execute(["git", "status", "--ignored"])
+        ignored_files = repo.git.execute(["git", "ls-files", "--exclude-standard", "--ignored", "--others"]).splitlines()
+        ignored_files = map(lambda x: x + "**" if x.endswith("/") else x, ignored_files)
```

Comment:
I believe this line is now unnecessary, directories should be included in ls-files. It should only list individual files that are ignored.

Reply:
I agree it could be removed now, and its counterpart on utils